pkg/apis/serving/v1alpha1/route_types.go

```diff
@@ -81,6 +81,10 @@ type TrafficTarget struct {
 	// Percent specifies percent of the traffic to this Revision or Configuration.
 	// This defaults to zero if unspecified.
 	Percent int `json:"percent"`
+
+	// URL displays the URL for accessing named traffic targets. URL is displayed in
+	// status. URL is disallowed on spec.
```

Comment:
URL must contain a scheme (e.g. http://) and a hostname, but nothing else (e.g. basic auth, url path).

Reply:
Updated comment